### DISCUSSION TITLE
Red text has low contrast on dark background [Edge Docs]

### DISCUSSION THREAD

**Discussion by MatheusRich** [rails/rails#51380] [Docs]
The contrast for red text (links, headings) is low on dark mode. See the devtools output below:

<img width="327" alt="image" src="https://github.com/rails/rails/assets/20938712/9fe7aed0-1402-420c-8f97-74bfdf6a9663">

We can either change the red shade (to something like `#ee181d`), but I think we're trying to maintain the Rails logo red share, right? In that case, I suggest updating the background color to something darker.

`#181818` might be the darker shade of gray I could get to pass on the [contrast checker](https://webaim.org/resources/contrastchecker/).

I'm opening a PR for this

**Comment by AmandaPerino** [comment]
Hi @MatheusRich we are going to work more on Dark Mode in the second phase of this update, which started as soon as the updates went live. In this phase we will most likely revert to a closer color scheme that we previously had, which will touch upon this issue as well as [this one](https://github.com/rails/rails/issues/51365) you created. 
Changes will most likely be made this week. 
Thanks for your feedback!

**Comment by AmandaPerino** [comment]
@MatheusRich Would you mind closing this issue since it is addressed in #51372 ?
